api/redhatopenshift/HcpCluster/hcpCluster-models.tsp

```diff
@@ -83,16 +83,13 @@ model ClusterSpec {
   api: ApiProfile;
 
   /** Enable FIPS mode for the cluster
-   * When set to true, `etcdEncryption` must be set to true
+   * When set to true, `etcdProfile.encryptionProfile.kms` must be set
    */
   @visibility("create")
   fips?: boolean = false;
 
-  /** Enables customer ETCD encryption, set during creation
-   * When set to true, `platform.etcdEncryptionSetId` must be set
-   */
-  @visibility("create")
-  etcdEncryption?: boolean = false;
+  /** Cluster etcd configuration */
+  etcd?: EtcdProfile;
```

Comment:
I added it as a top-level field of the ClusterSpec.
An alternative would be within the "platform" section.

Let me know your thoughts about it.